examples/Example 1.yml

```diff
@@ -0,0 +1,58 @@
+# Teams Section
+Teams:
+- Name: Team 1
+  Identity: Security Group 1
+- Name: Team 2
+  Identity: Security Group 2
+
+# Patterns
+Patterns:
+- !Security # Exclamation mark is not actual yaml, but the yaml library understands it as inheritance, parent is pattern
+  Name: Default
+  Application Groups:
+  - Name: Contributor
+    Members: Security Group {0} # {0} will be replaced with a key using a string.format
+    Namespaces:
+    - Name: Project
+      Allow:
+      - Create tag definition
+      - Create test runs
+      - Edit build quality
+      - Edit project-level information
+      - Manage project properties
+      - Manage test configurations
+      - Manage test environments
+      - Move work items out of this project
+      - Start a build
+      - Suppress notifications for work item updates
+      - View analytics
+      - View project-level information
+      - View test runs
+      - Write to build operational store
+      Deny:
+      - Bypass rules on work item updates
+      - Change process of team project.
+      - Delete and restore work items
+      - Delete team project
+      - Delete test runs
+      - Update project visibility
+      - Write to build operational store
+      Not Set:
+      - ...
+
+# Team Projects
+Team Projects:
+- Key: 00001 # Key is used to generate the security group
+  Name: Team Project 1
+  Patterns: 
+  - !Security
+    Name: Default # References the Default security pattern
+  Agent Pools:
+  - linux
+  - windows
+  Status: Active # Active, Archived (meaning still available but readonly), Removed.
+  Teams:
+  - Team 1 # References Team 1 from the teams section
+
+# Mappings
+00001: b939b232-23fe-4aa3-8c56-9e24fd199cb3 # Mapping key to actual team project identifier.
```

Comment:
Shouldn't mappings be their own section ?

Reply:
I agree. I think all the section(s) should be in their own files for readability.

Reply:
What is your opinion? one file or multiple files?